frontend/pages/settings/dlna.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <settings-page page-title="settings.dlna.dlna">
+    <template #actions>
+      <v-btn
+        rel="noreferrer noopener"
+        href="https://jellyfin.org/docs/general/networking/dlna.html"
+        target="_blank"
+      >
+        {{ $t('settings.help') }}
+      </v-btn>
+    </template>
+    <template #content>
+      <v-col cols="12" md="6" lg="5" class="py-4">
+        <v-card class="mb-4">
+          <v-list two-line flat>
+            <v-list-item-group>
+              <v-list-item
+                v-for="dlnaSettingName in Object.keys(bindToRelevantConfig)"
+                :key="dlnaSettingName"
+              >
+                <v-list-item-content>
+                  <v-list-item-title
+                    v-text="$t('settings.dlna.' + dlnaSettingName + '.text')"
+                  />
+                  <v-list-item-subtitle
+                    v-text="
+                      $t('settings.dlna.' + dlnaSettingName + '.description')
+                    "
+                  />
+                </v-list-item-content>
+                <v-list-item-action>
+                  <v-switch
+                    v-if="typeof dlnaSettings[dlnaSettingName] === 'boolean'"
```

Comment:
Why we need typeof here? Do we know if the API can return something else that it's not a boolean?

Reply:
We do need typeof here as we want to display different control elements (Switches for boolean values, Textfields for strings, Selectors for Enum based values) based on which type the data has we want to control.

This seemed the best approach from my perspective as it allows us to loop through all relevant data without needing to care which exact control we want to display for each field.